src/clientversion.cpp

```diff
@@ -99,5 +100,7 @@ std::string FormatSubVersion(const std::string& name, int nClientVersion, const
         ss << ")";
     }
     ss << "/";
+    if (!fBaseNameOnly && IsArgSet("-bip148"))
+        ss << "UASF-Segwit:0.3(BIP148)/";
```

Comment:
IMO it would be more technically-correct to simply add "BIP148 Enforcing" to the uacomments by default, when this flag is set.